SingularityService/src/main/java/com/hubspot/singularity/data/StateManager.java

```diff
@@ -133,6 +134,10 @@ public void save(SingularityHostState hostState) throws InterruptedException {
       numTasks.incr(pendingTaskId.getRequestId());
     }
 
+    for (SingularityTaskId cleaningTaskId : taskManager.getCleanupTaskIds()) {
+      numTasks.decr(cleaningTaskId.getRequestId());
```

Comment:
Do we still want to include cleaning when checking for under provisioned, this will exclude it for both under _and_ over? Trying to make sure we avoid any false alerts if possible